<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AliasEdit</class>
 <widget class="QDialog" name="AliasEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>551</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit Alias</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="reactions_pattern" native="true">
     <layout class="QHBoxLayout" name="_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="Pattern_label">
        <property name="text">
         <string>Alias:</string>
        </property>
        <property name="buddy">
         <cstring>Pattern</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Pattern"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="Text_label">
     <property name="text">
      <string>&amp;Send:</string>
     </property>
     <property name="buddy">
      <cstring>Text</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="Text">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_main">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="_options">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <layout class="QFormLayout" name="_send">
         <item row="0" column="0">
          <widget class="QLabel" name="UserSendTo_label">
           <property name="text">
            <string>Send to:</string>
           </property>
           <property name="buddy">
            <cstring>UserSendTo</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="UserSendTo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string>World</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Command</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Output</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Status</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Notepad (new)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Notepad (append)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Notepad (replace)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Log file</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Variable</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Script</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="Sequence_label">
           <property name="text">
            <string>Seq&amp;uence:</string>
           </property>
           <property name="buddy">
            <cstring>Sequence</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="Sequence">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimum">
            <number>-30000</number>
           </property>
           <property name="maximum">
            <number>30000</number>
           </property>
           <property name="value">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="Script_label">
           <property name="text">
            <string>Script:</string>
           </property>
           <property name="buddy">
            <cstring>Script</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="Script"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="Group_label">
           <property name="text">
            <string>Group:</string>
           </property>
           <property name="buddy">
            <cstring>Group</cstring>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="Group"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="Variable_label">
           <property name="text">
            <string/>
           </property>
           <property name="buddy">
            <cstring>Variable</cstring>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="Variable">
           <property name="visible">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="Label"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="Label_label">
           <property name="text">
            <string>&amp;Label:</string>
           </property>
           <property name="buddy">
            <cstring>Label</cstring>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>2</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="_checkboxes">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QCheckBox" name="Enabled">
         <property name="text">
          <string>En&amp;abled</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="OmitFromOutput">
         <property name="text">
          <string>O&amp;mit from output</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="OmitFromLog">
         <property name="text">
          <string>&amp;Omit from log file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="OneShot">
         <property name="text">
          <string>One-shot</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="Temporary">
         <property name="text">
          <string>Temporary</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="IgnoreCase">
         <property name="text">
          <string>&amp;Ignore case</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="KeepEvaluating">
         <property name="text">
          <string>&amp;Keep evaluating</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="ExpandVariables">
         <property name="text">
          <string>Expand variables</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="LowercaseWildcard">
         <property name="text">
          <string>Make wildcards lower-case</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="IsRegex">
         <property name="text">
          <string>Regular expression</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="EchoAlias">
         <property name="text">
          <string>Echo alias</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="OmitFromCommandHistory">
         <property name="text">
          <string>Omit from command history</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="Menu">
         <property name="text">
          <string>Menu</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="Repeats">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Repeat on same line</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>2</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AliasEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>917</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AliasEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>917</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
